core/src/main/resources/hudson/model/Messages.properties

```diff
@@ -177,6 +177,7 @@ Item.CONFIGURE.description=Change the configuration of a job.
 Item.READ.description=See a job. (You may deny this permission but allow Discover to force an anonymous user to log in to see the job.)
 Item.RENAME.description=Rename a job.
 ItemGroupMixIn.may_not_copy_as_it_contains_secrets_and_=May not copy {0} as it contains secrets and {1} has {2}/{3} but not /{4}
+JDK.DisplayName=JDK
```

Comment:
is this actually localise-able though?

I don't know if it actually makes sense for this.

Reply:
Thanks for your comment @timja. I mostly agree that it doesn't make a lot of sense, but this label appears in the UI (Global Tool Configuration) alongside with other labels which are localized using the same pattern. Most likely for this reason it was marked as 'Todo'. In any case such very small improvement will make the world slightly better. Additionally this fix suitable for first commit and help me to learn processes.

Reply:
No objections from me. Better to localize